### ISSUE TITLE
Extreme lag when looking up recipies with ME (wireless) terminal

### ISSUE THREAD

**Issue by Schlauchiboy** [AppliedEnergistics/Applied-Energistics-2#7563]
### How to reproduce the performance issue

I removed all the cables from the controller (7x7x7) to my devices to see if there was a problem. But i am only running one cable from controller to terminal and problem is still there. When i open the ME terminal everything is fine but when i press on an item to look up the things i can craft (JEI) with this item my FPS drop to 1-2 FPS. When i open my inventory normally with "E" and look up the same recipe i have like 150-160 FPS. I am playing ATM9.

### Spark profile link

https://spark.lucko.me/Xlcz9oPwh9

### Which minecraft version are you using?

1.20

### On which mod loaders does it happen?

Forge

### Additional details

AE2 version: 15.0.19
Forge Version: 47.2.19

**Comment by Mari023** [comment]
weird, the new link works for me...
![graphic](https://github.com/AppliedEnergistics/Applied-Energistics-2/assets/38946771/8159617b-f7c0-42df-8a29-026a1d0ffe7f)

**Comment by Schlauchiboy** [comment]
yes that is my data. I don´t know what this means but is there anything special that is related to my issue?

**Comment by Peri-42** [comment]
I'm having the same issue, same modpack and version. it seems related to #7404 and #7356, here's my spark profile: https://spark.lucko.me/JesEBW4C3i

**Comment by KuroPeach** [comment]
same question
https://spark.lucko.me/rvErCJ5kIW
